src/hotspot/share/gc/shared/stringdedup/stringDedupThread.hpp

```diff
@@ -40,6 +40,9 @@
 class StringDedupThread : public JavaThread {
   friend class VMStructs;
 
+  // Perf data for CPU time consumed by the string dedup thread.
+  static PerfVariable* _g1_concurrent_dedup_thread_cpu_time;
```

Comment:
This variable seems unused. ??

Reply:
It's better to move this variable to stringDedupProcessor.hpp/cpp. Then it will be updated in `StringDedup::Processor::run()`.

Also, we should rename this variable so it is not specific to G1. Shenandoah also supports string dedup (https://bugs.openjdk.org/browse/JDK-8203641).